Symfony/CS/Tests/Fixer/Symfony/YodaConditionsFixerTest.php

```diff
@@ -0,0 +1,130 @@
+<?php
+
+/*
+ * This file is part of the PHP CS utility.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\CS\Tests\Fixer\Symfony;
+
+use Symfony\CS\Tests\Fixer\AbstractFixerTestBase;
+
+/**
+ * @author Bram Gotink <bram@gotink.me>
+ * @author Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ */
+class YodaConditionsFixerTest extends AbstractFixerTestBase
+{
+    /**
+     * @dataProvider provideExamples
+     */
+    public function testFixer($expected, $input = null)
+    {
+        $this->makeTest($expected, $input);
+    }
+
+    public function provideExamples()
+    {
+        return array(
+            // valid conditions that should not be changed
+            array('<?php return array(2) == $a;'),
+            array('<?php return $a == array(2);'),
```

Comment:
Hi!
This should be 2 fixers. What you want will be sth like ExtendedYodaFixer, what we have some time ago. That fixer won't be applied on Symfony. Please look at commit history of this PR, the first one is a revert of removing a fixer.
The second one will not change this and will be sth like SimplifiedYodaFixer. This fixer will be applied on Symfony.